controllers/profiles/common/persistence.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Apache Software Foundation (ASF)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+
+	operatorapi "github.com/apache/incubator-kie-kogito-serverless-operator/api/v1alpha08"
+)
+
+const (
+	defaultSchemaName = "default"
+)
+
+func ConfigurePersistence(serviceContainer *corev1.Container, options *operatorapi.PersistenceOptions, defaultSchema, namespace string) *corev1.Container {
+	c := serviceContainer.DeepCopy()
+	if options != nil {
+		if options.PostgreSql != nil {
+			c.Image = DataIndexImageBase + PersistenceTypePostgressql
+			c.Env = append(
+				c.Env,
+				configurePostgreSqlEnv(options.PostgreSql, defaultSchema, namespace)...,
+			)
+		}
+	}
+	return c
+}
+
+func configurePostgreSqlEnv(postgresql *operatorapi.PersistencePostgreSql, databaseSchema, databaseNamespace string) []corev1.EnvVar {
```

Comment:
Can you move these hardcoded strings and integers to a private const in this file?